homeassistant/components/generic/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
since you are using `av` in the config flow, it needs to be added as a dependency here.